finalPrep/README.md

```diff
@@ -32,23 +32,29 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
```

Comment:
I don't see your calendar in this file - let me know if I'm missing, if not, please add!

Reply:
![calendar screenshot 1](https://user-images.githubusercontent.com/36666973/147839345-bd4a9124-8463-43e2-97c3-27ed01192b4e.png)

@ameseee here is a screenshot! Sorry I just am now getting to this, been gone for the holidays.

Reply:
Thanks Matt. It looks like this puts you at 44 hours/week dedicated to school - probably not realistic. I'd encourage you to go back through your notes/Session 1 and consider both the demand of Turing and your personal needs outside of Guitar (meal prep, errands, family, etc).